### PR TITLE
[CI] use obltGitHubComments

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Use https://github.com/elastic/apm-pipeline-library/blob/master/vars/obltGitHubComments.txt

## Why is it important?

Avoid hardcoded strings and easy to normalise the GitHub comments in all the oblt projects

You can also see in the GitHub comment that there is a specific section for **some** of those GitHub comments called `🤖 GitHub comments`

### CODE REVIEW

**Comment by trentm** [comment]
I've given `observablt-robots` admin access to this repo. Perhaps that is what is holding this up.

@v1v Do you know what "approval-list/elastic/ecs-logging-nodejs.yml" is? It is mentioned in the details of the "Steps errors" above. I see https://github.com/elastic/apm-pipeline-library/tree/master/resources/approval-list/ has a list of some repos, but not this one. However, are those files just early tests of something that isn't used, because all those files only list your account.

**Review by trentm — APPROVED** [review]

**Comment by v1v** [comment]
> @v1v Do you know what "approval-list/elastic/ecs-logging-nodejs.yml" is? It is mentioned in the details of the "Steps errors" above. I see https://github.com/elastic/apm-pipeline-library/tree/master/resources/approval-list/ has a list of some repos, but not this one. However, are those files just early tests of something that isn't used, because all those files only list your account.

That's some unused internal approval list implementation to give granted access to run in the CI to certain users or a project based. It was implemented but never activated, likely there is a missing condition if the file exists, good catch, it should not be shown in the PR comment at all. I'll add the fix to avoid those misleading comments, that only happens when the contributor doesn't have write access to the project.

**Pull request merged**
